### PR TITLE
Fix mentor dashboard page routes

### PR DESCRIPTION
**Pull request by Isha988**
## Description of Changes

Fix the mentor dashboard on refresh error
Created new page scheme
Changed Sidebar according to new scheme

### CODE REVIEW

File: pages/dashboard/mentor/calendar.page.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+function calendar() {
+  return (
+    <>
+      <Sidebar />
+      <Calendar />
+    </>
+  );
+}
+
+export default calendar;
```
Comment (anmode) [inline pages/dashboard/mentor/calendar.page.js:14]: Oh i was thinking to render components on the basis on hash in url....like
url m `currentTab = profile`  h toh profile component render ho..
I mean aise kaafi page.js banane pdenge.

**Review by anmode — CHANGES_REQUESTED** [review]
Need some clarity

File: pages/dashboard/mentor/calendar.page.js (hunk 1 above)
Comment (Isha988) [reply pages/dashboard/mentor/calendar.page.js:14]: > I mean aise kaafi page.js banane pdenge.

I agree, so okay we will go with your approach

**Comment by Isha988** [comment]
Done 👍

**Review by anmode — APPROVED** [review]

**Comment by anmode** [comment]
Liked the changes!

**Comment by Isha988** [comment]
> Liked the changes!

Thank you 
Just a gentle reminder to add level label

**Comment by anmode** [comment]
Oh sorry! Actually i have switched off notifications for sometime of frontend repo.

**Pull request merged**